docs/release.rst

```diff
@@ -278,10 +278,8 @@ automatically for all `listed <https://github.com/cockpit-project/bots/blob/main
         'master': [
             'fedora-35/rawhide',
         ],
-        'fedora-38': [
-            'fedora-38',
-        ],
-        '_manual': [
+        'fedora-39': [
+            'fedora-39',
```

Comment:
I don't think we want to adjust this each branching.

Reply:
...`fedora-NN` ?